pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -302,14 +302,15 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(pInfo *framework.QueuedPodI
 		return fmt.Errorf("pod: %v is already present in unschedulable queue", nsNameForPod(pod))
 	}
 
-	// Refresh the timestamp since the pod is re-added.
-	pInfo.Timestamp = p.clock.Now()
 	if _, exists, _ := p.activeQ.Get(pInfo); exists {
 		return fmt.Errorf("pod: %v is already present in the active queue", nsNameForPod(pod))
 	}
 	if _, exists, _ := p.podBackoffQ.Get(pInfo); exists {
 		return fmt.Errorf("pod %v is already present in the backoff queue", nsNameForPod(pod))
 	}
+	
+	// Refresh the timestamp since the pod is re-added.
+	pInfo.Timestamp = p.clock.Now()
 
```

Comment:
lgtm, but I cannot see what's the real impact, the pods in the activeQ will still be popped up with the original order.

Reply:
If the Pod in activeQ have the same priority, they will be FIFO according to the enqueue time.
If the enqueue time is refreshed, the order in the activeQ should be updated. 
So I don’t think should refresh the enqueue time